bitcoin/src/bip32.rs

```diff
@@ -428,7 +428,7 @@ impl fmt::Debug for DerivationPath {
 pub type KeySource = (Fingerprint, DerivationPath);
 
 /// A BIP32 error
-#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
```

Comment:
Should we be removing things?

Reply:
I vote yes, for consistency's sake. Though I think if we're gonna make `PartialEq` and `Eq` standard (debatable, see my previous comment) we should have `Hash` as well.